.env.sample

```diff
@@ -6,6 +6,10 @@ LEGACY_JWT_SECRET=legacy_jwt_secret
 AUTH_JWT_SECRET=auth_jwt_secret
 AUTH_JWT_TTL=60
 
+ENCRYPTION_SERVER_KEY=change-me-!
+ENCRYPTION_SALT_LENGTH=64
+ENCRYPTION_IV_LENGTH=16
```

Comment:
These definitely shouldn't be env variables. These should be hardcoded based on the encryption version of the protocol. So if anything you'd have these as consts somewhere. Take a look at this: https://github.com/standardnotes/snjs/blob/master/packages/snjs/lib/protocol/operator/algorithms.ts

Reply:
Can you send me some links to where it says that salt and iv have to have a fixed length when using AES-GCM ?

Reply:
https://crypto.stackexchange.com/a/42414